templates/Registration/recovery_token.html.twig

```diff
@@ -0,0 +1,41 @@
+{% extends 'base.html.twig' %}
+
+{% block subtitle %}{{ "registration.recovery-token-headline"|trans }}{% endblock %}
+
+{% form_theme form 'Form/fields.html.twig' %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-xs-12 col-sm-6 col-md-6">
+            <h3>{{ "recovery-token.headline"|trans }}</h3>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-xs-12 col-sm-6 col-md-6">
+            <p class="lead">{{ "recovery-token.created-headline"|trans }}</p>
+            <p class="lead alert alert-warning">{{ recovery_token }}</p>
```

Comment:
This should be only a warning (yellow) when the user don't have a recovery_token, otherwise it should be green.

Reply:
Agreed, fixed now.